cli.sh

```diff
@@ -2178,12 +2337,30 @@ config_global_shared_hook_repos() {
             fi
         done
 
-        git config --global githooks.shared "$NEW_LIST"
+        git config "$SHARED_TYPE" githooks.shared "$NEW_LIST"
+
     elif [ "$1" = "reset" ]; then
-        git config --global --unset githooks.shared
+        SHARED_TYPE="--global"
+        if echo "$2" | grep -qE "\-\-(local)"; then
+            SHARED_TYPE="$2"
+        elif [ -n "$2" ]; then
+            manage_configuration "help"
+            echo "! Wrong argument $($2)" >&2
```

Comment:
Are you sure about the `$($2)` bit? Should this be
```suggestion
            echo "! Wrong argument \`$2\`" >&2
```

Reply:
Thx! Typo